pkg/git/git_test.go

```diff
@@ -147,20 +156,105 @@ func Test_validateGitAuth(t *testing.T) {
 			credsDir, cleanup := createTempDir(t)
 			defer cleanup()
 			if tt.wantSSHdir {
-				os.MkdirAll(credsDir+"/.ssh", 0755)
+				err := os.MkdirAll(filepath.Join(credsDir, "/.ssh"), fileMode)
```

Comment:
I think the `/` in `"/.ssh"` might not be needed because `filepath.Join` will add it. (similar comment for other instances below)